test/tests/integrationTests.js

```diff
@@ -262,8 +262,9 @@ describe("Zotero.Integration", function () {
 					displayAlertStub.restore();
 				});
 			
-				describe('when the style is not from zotero.org or citationstyles.org', function() {
+				describe('when the style is not from a trusted source', function() {
```

Comment:
As described in the 3 following tests. Prompt if untrusted source, install without prompt otherwise.